src/python/pants/backend/python/subsystems/ipython.py

```diff
@@ -25,7 +25,7 @@ class IPython(PythonToolBase):
     options_scope = "ipython"
     help = "The IPython enhanced REPL (https://ipython.org/)."
 
-    default_version = "ipython==7.16.1"  # The last version to support Python 3.6.
+    default_version = "ipython>=7.34,<8"  # ipython 8 does not support Python 3.7.
```

Comment:
Is dropping Python 3.6 something that we'll want to cherrypick, or should the docs edit and the version bump happen separately?

Reply:
We dropped Python 3.6 support in 2.11 with https://github.com/pantsbuild/pants/pull/14803, so this is safe to cherry-pick.